src/pytypes/onefuzztypes/events.py

```diff
@@ -121,14 +121,17 @@ class EventPoolCreated(BaseEvent):
 
 class EventProxyCreated(BaseEvent):
     region: Region
+    proxy_id: UUID
 
 
 class EventProxyDeleted(BaseEvent):
     region: Region
+    proxy_id: UUID
 
 
 class EventProxyFailed(BaseEvent):
     region: Region
+    proxy_id: UUID
```

Comment:
Similar to EventProxyFailed.proxy_id, this needs to be optional, otherwise this is a breaking change for those using our python library to parse events.

Reply:
Done.